photon-ml/src/integTest/scala/com/linkedin/photon/ml/DriverIntegTest.scala

```diff
@@ -54,15 +55,18 @@ class DriverIntegTest extends SparkTestUtils with TestTemplateWithTmpDir {
     appendCommonJobArgs(args, tmpDir)
     args += CommonTestUtils.fromOptionNameToArg(OPTIMIZER_TYPE_OPTION)
     args += "TRON"
+    args += CommonTestUtils.fromOptionNameToArg(MAX_NUM_ITERATIONS_OPTION)
+    args += "10"
 
     MockDriver.runLocally(args = args.toArray,
       expectedStages = Array(DriverStage.INIT, DriverStage.PREPROCESSED, DriverStage.TRAINED),
-      expectedNumFeatures = 14, expectedNumTrainingData = 250, expectedIsSummarized = false)
+      expectedNumFeatures = EXPECTED_NUM_FEATURES, expectedNumTrainingData = EXPECTED_NUM_TRAINING_DATA,
```

Comment:
Method invocations that cannot fit on one line should have only one argument per line:
http://docs.scala-lang.org/style/indentation.html#methods_with_numerous_arguments
Same goes for all MockDriver.runLocally(...) calls